wiki/Tournaments/GBC/GBC_2022_Spring/en.md

```diff
@@ -0,0 +1,208 @@
+12. If less than the minimum amount of required players are present at match time, it can be postponed for up to 10 minutes. If, after this period, there are still not enough players for either, a *win by default* will be declared for the team with the most members present.
+    - The minimum amount of required players is defined as the amount of players needed to play a beatmap.
+13. Exchanging players during games is allowed without limitations, except tournament special rules.
+    - see Match instructions for detail.
+14. Players are expected to keep the match running fluently and without delays. Excessive match delays from the players' side may result in penalties being applied.
+15. If a player disconnects between beatmaps and the team cannot provide a substitute, the match can be delayed for up to 10 minutes (limited to once per team, per match). 
+16. All players and staff must be treated with respect. Instructions of the referees and the Host are to be followed. Decisions labeled as final are not to be objected.
+17. Disrupting the match by foul play, insulting and provoking other players or staff, delaying the match or other deliberate inappropriate misbehaviour are strictly prohibited.
+18. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms, too.
+    - Breaking the chat rules results in a silence. Silenced players cannot participate at multiplayer matches and must be exchanged for the time being.
+19. Unexpected incidences will be handled by the Host. Referees may allow higher tolerance depending on the given circumstances. This is up to their discretion.
+20. Penalties for violating the tournament rules include, but are not limited to:
+    - Exclusion of specific players for one beatmap.
+    - Exclusion of specific players for an entire match.
+    - Declaring the match as forfeited, or as a win by default for the other team.
+    - Disqualification from the entire tournament.
+    - Disqualification from the current and future GBC series.
+    - Disqualification from the current and future official & commnunity tournaments, until appealed.
+21. The Host reserves the right to modify these rules at any moment. Any such canges will be announced in advance.
+
+### Tournament registration
+
+1. Please send the Host **a private message** and **fill [this form](https://wj.qq.com/s2/9902248/95cf/)** to register. 
+2. Each player could send a VOD clips or an in-game replays to Host, which will be used on Draft phase. 
+3. To ensure valid and serious registrations, every registered user will be manually checked by the Host.
+   - To be successfully registered, a player must not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
+   - **No rank limitations for this tournament.** host have the right to choose several best players to participate in showmatch (showcase by a match) **instead of** the actual tournament for balance issues.
```

Comment:
Orgional Sentence:
>    - **No rank limitations for this tournament.** but host have the rights to choose several **best** players may invited to **pricipate showmatch instead of actual tournament** for balance issues.

From TicClick:

> I don't understand this sentence, what were you trying to explain?

Response: 
no lowest/highest rank limitation for players. if amounts of registered players are higher than tournament limits, some top players (not fully equal to have highest PP or world ranking) will be removed from the player list.
but this tournament has show matches between each stage, these players removed by above reason will be invited into such show match.

Reply:
if you agree with the host's explain, i will fixed this sentence by using later one suggestion.

Reply:
seems ok, but put a hyphen in "high-level registrants" (i've edited the suggestion)